### PR TITLE
fix(app): solve nested folder structure in pages

### PR DESCRIPTION
**Pull request by koji**
# Overview
solve nested folder structure in pages and add test.
remove `OnDeviceDisplay` folder
I noticed that we are missing a few page components' tests. I will follow up them in another PR.
Update a few tests that VSCode shows warnings with what Brian told us last week.

```ts
const [{ getByText }] = render()
getByText('something')

render()
screen.getByText('something') 

// click() -> fireEvent.click(targetElement)
```

# Test Plan

# Changelog
- move components under `pages/OnDeviceDisplay` to `pages`
- update paths in tests
- remove `OnDeviceDisplay` folder

# Review requests
Do we want to keep `ODDTempMenu`?

# Risk assessment
low

### CODE REVIEW

**Review by shlokamin — COMMENTED** [review]
im not sure that we want to change local imports from within a module to import from the public (external) package name. and we certainly don't want to import from the public package AND reach into private directories. 

i.e. from within the app, we shouldn't be doing something like this:

`import { ConnectViaEthernet } from '@opentrons/app/src/pages/ConnectViaEthernet'`

should either be `import { ConnectViaEthernet } from '@opentrons/app'` or a relative import. and i'd prefer a relative import because, well, we're importing from within the same package!

how come we're not going with relative imports anymore?

**Comment by koji** [comment]
> im not sure that we want to change local imports from within a module to import from the public (external) package name. and we certainly don't want to import from the public package AND reach into private directories.
> 
> i.e. from within the app, we shouldn't be doing something like this:
> 
> `import { ConnectViaEthernet } from '@opentrons/app/src/pages/ConnectViaEthernet'`
> 
> should either be `import { ConnectViaEthernet } from '@opentrons/app'` or a relative import. and i'd prefer a relative import because, well, we're importing from within the same package!
> 
> how come we're not going with relative imports anymore?

I think the paths were changed by VSCode, I will revert them.

File: app/src/pages/TempODDMenu/index.tsx [hunk 1]
```diff
@@ -4,17 +4,17 @@ import { useDispatch, useSelector } from 'react-redux'
...
   startRobotUpdate,
-} from '../../../redux/robot-update'
-import { getLocalRobot } from '../../../redux/discovery'
+} from '../../redux/robot-update'
+import { getLocalRobot } from '../../redux/discovery'
 
-import type { RouteProps } from '../../../App/types'
+import type { RouteProps } from '../../App/types'
 
 export function TempODDMenu(): JSX.Element {
```
Comment (brenthagen) [inline app/src/pages/TempODDMenu/index.tsx]: also probably time to delete the temp menu 😃

**Review by brenthagen — APPROVED** [review]
changes make sense to me, smoke tested the app without issue

File: app/src/pages/TempODDMenu/index.tsx (hunk 1 above)
Comment (koji) [reply app/src/pages/TempODDMenu/index.tsx]: I will delete that.

**Pull request merged**